homeassistant/components/homematicip_cloud/sensor.py

```diff
@@ -79,6 +80,8 @@
         ):
             entities.append(HomematicipTemperatureSensor(hap, device))
             entities.append(HomematicipHumiditySensor(hap, device))
+        elif isinstance(device, (AsyncRoomControlDeviceAnalog,),):
```

Comment:
Is the second comma necessary? Just comparing this with line 67.
```suggestion
        elif isinstance(device, (AsyncRoomControlDeviceAnalog,)):
```